validation/barotropic_gyre/barotropic_gyre.jl

```diff
@@ -0,0 +1,124 @@
+# # Barotropic gyre
+
+using Oceananigans
+using Oceananigans.Grids
+
+using Oceananigans.Coriolis:
+    HydrostaticSphericalCoriolis,
+    VectorInvariantEnergyConserving,
+    VectorInvariantEnstrophyConserving
+
+using Oceananigans.Models.HydrostaticFreeSurfaceModels:
+    HydrostaticFreeSurfaceModel,
+    VectorInvariant,
+    ExplicitFreeSurface
+
+using Oceananigans.TurbulenceClosures: HorizontallyCurvilinearAnisotropicDiffusivity
+using Oceananigans.Utils: prettytime, hours
+using Oceananigans.OutputWriters: JLD2OutputWriter, TimeInterval, IterationInterval
+
+using Statistics
+using JLD2
+using Printf
+using GLMakie
+
+Nx = 120
+Ny = 60
+
+# A spherical domain
+grid = RegularLatitudeLongitudeGrid(size = (Nx, Ny, 1),
+                                    longitude = (-30, 30),
+                                    latitude = (-15, 45),
```

Comment:
I see that this jet overlaps the equator.  Wouldn't it be easier to go from `+15` to `+45`?  As we know, geostrophic balance breaks down at the equator and this seems like a harder problem than a sub-tropical gyre.